File: app/src/main/java/com/cool/eye/mvp/java/test/TestFragment.java
package com.cool.eye.mvp.java.test;


import com.cool.eye.mvp.java.BaseMvpFragment;
import com.cool.eye.mvp.java.LoadView;

/**
 * @author ycb
 * @date 2018/8/17
 */
public class TestFragment extends BaseMvpFragment<TestPresenter> implements  ITestView{

    @Override
    public TestPresenter getPresenter() {
        return new TestPresenter(this);
    }

    @Override
    public void setTest(String testMsg) {

    }

    @Override
    public void toAnotherPage() {

    }

    @Override
    public LoadView getLoadView() {
        return LoadView.get(getContext());
    }
}

File: app/src/main/java/com/cool/eye/mvp/kotlin/test/TestFragment.kt
package com.cool.eye.mvp.kotlin.test


import android.content.Context
import com.cool.eye.mvp.kotlin.BaseMvpFragment

/**
 * @author ycb
 * @date 2018/8/17
 */
class TestFragment : BaseMvpFragment<TestPresenter>(), ITestView {

    override fun getPresenter(): TestPresenter {
        return TestPresenter(this)
    }

    override fun setTest(testMsg: String) {

    }

    override fun toAnotherPage() {

    }

    override fun withContext(): Context {
        return context
    }
}
